### PR TITLE
Remove config file

### PR DESCRIPTION
**Pull request by stdevMac**
## Changes:
- Remove config file and all code dependecies

## Types of changes

What types of changes does your code introduce?
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update
- [x] Refactoring (no functional changes, no api changes)

### CODE REVIEW

File: internal/pkg/clients/clients.go [hunk 1]
```diff
@@ -89,17 +88,10 @@ func (c ClientInfo) Clients(clientTypes []string) (clients OrderedClients, errs
...
-			continue
-		}
-		log.Debugf(configs.ConfigClientsMsg, clientType, strings.ToLower(strings.Join(configClients, ", ")))
-
-		for _, client := range configClients {
+		for _, client := range supportedClients {
 			// Check if the client is supported
-			supported := utils.Contains(supportedClients, client)
-			clients[clientType][client] = Client{Name: client, Type: clientType, Supported: supported}
+			//supported := utils.Contains(supportedClients, client)
```
Comment (AntiD2ta) [inline internal/pkg/clients/clients.go:93]: Remove this comment

File: internal/pkg/clients/clients_test.go [hunk 2]
```diff
@@ -108,9 +92,9 @@ func TestClients(t *testing.T) {
 		{},
```
Comment (AntiD2ta) [inline internal/pkg/clients/clients_test.go:92]: Add a test case for Gnosis, as it has different supported execution clients and consensus than mainnet

File: docs/docs/commands/version.mdx [hunk 3]
```diff
@@ -21,17 +21,13 @@ Flags:
...
+      --logLevel string   Set Log Level (default "info")
 ```
 
 ## Execution Example
 
 ```
 $ sedge version
-Initializing configuration
-Using config file: /Users/maceo/.sedge.yaml
-2022-12-13 10:15:30 -- [INFO] [Logger Init] Logging configuration: {Level:info}
```
Comment (AntiD2ta) [inline docs/docs/commands/version.mdx:34]: Why the `[Logger Init]` got removed?

File: docs/docs/commands/networks.mdx [hunk 4]
```diff
@@ -21,17 +21,13 @@ Flags:
...
+      --logLevel string   Set Log Level (default "info")
 ```
 
 ## Execution Example
 
 ```
 $ sedge networks
-Initializing configuration
-Using config file: /Users/maceo/.sedge.yaml
-2022-12-13 10:13:02 -- [INFO] [Logger Init] Logging configuration: {Level:info}
```
Comment (AntiD2ta) [inline docs/docs/commands/networks.mdx:34]: Why did the [Logger Init] got removed?

File: docs/docs/commands/logs.mdx [hunk 5]
```diff
@@ -34,8 +34,6 @@ The execution of `sedge down` will close and remove all the opened containers an
 
 ```
 $ sedge down
-Using config file: /home/maceo/.sedge.yaml
-2022-12-12 16:54:24 -- [INFO] [Logger Init] Logging configuration: {Level:info}
```
Comment (AntiD2ta) [inline docs/docs/commands/logs.mdx:38]: Why did the [Logger Init] got removed?

File: docs/docs/commands/keys.mdx [hunk 6]
```diff
@@ -61,9 +61,6 @@ The `keys` command will first check if Docker is installed (because use docker t
 The next step involves a prompt for an Eth1 address to be used to create the [withdrawal credentials](/docs/concepts/keystore). If you don't provide one, then the tool will generate withdrawal credentials with the mnemonic-derived withdrawal public key.
 
 ```logs
-Initializing configuration
-Using config file: /Users/teno/.sedge.yaml
-2022-XX-XX 00:00:00 -- [INFO] [Logger Init] Logging configuration: {Level:info}
```
Comment (AntiD2ta) [inline docs/docs/commands/keys.mdx:66]: Why did the [Logger Init] got removed?

File: docs/docs/commands/down.mdx [hunk 7]
```diff
@@ -30,8 +30,6 @@ The execution of `sedge down` will close and remove all the opened containers an
 
 ```
 $ sedge down
-Using config file: /home/maceo/.sedge.yaml
-2022-12-12 16:54:24 -- [INFO] [Logger Init] Logging configuration: {Level:info}
```
Comment (AntiD2ta) [inline docs/docs/commands/down.mdx:34]: Why did the [Logger Init] got removed?

File: docs/docs/commands/clients.mdx [hunk 8]
```diff
@@ -51,16 +48,7 @@ Using config file: /Users/maceo/.sedge.yaml
  3   -                   prysm               teku
```
Comment (AntiD2ta) [inline docs/docs/commands/clients.mdx:48]: Remove `prysm`

File: docs/docs/commands/clients.mdx [hunk 9]
```diff
@@ -30,19 +30,16 @@ The execution of `sedge clients` will result in an output like this, that will s
 
 ```
 $ sedge clients
-Initializing configuration
-Using config file: /Users/maceo/.sedge.yaml
-2022-12-08 08:57:47 -- [INFO] [Logger Init] Logging configuration: {Level:info}
```
Comment (AntiD2ta) [inline docs/docs/commands/clients.mdx:35]: Why did the [Logger Init] got removed?

File: docs/docs/commands/cli.mdx [hunk 10]
```diff
@@ -62,14 +61,8 @@ Global Flags:
 
 ```logs
 $ ./sedge cli
-
-Initializing configuration
-Config File ".sedge" Not Found in "[/home/sedge]"
-Config file not found on the path provided nor in the home directory
-# highlight-next-line
-Generating config file in the /home/sedge directory
-2022-XX-XX 00:00:00 -- [INFO] [Logger Init] Logging configuration: {Level:info}
```
Comment (AntiD2ta) [inline docs/docs/commands/cli.mdx:71]: Why did the [Logger Init] got removed?

File: cli/root.go [hunk 11]
```diff
@@ -114,24 +70,14 @@ returns :-
 none
 */
 func initLogging() {
-	var config configs.LogConfig
-
 	log.SetFormatter(&nested.Formatter{
 		HideKeys:        true,
-		FieldsOrder:     []string{configs.Component},
```
Comment (AntiD2ta) [inline cli/root.go:121]: ???

File: cli/root.go [hunk 12]
```diff
@@ -114,24 +70,14 @@ returns :-
...
-	if err != nil {
-		log.WithField(configs.Component, "Logger Init").Errorf("Unable to decode into struct, %v", err)
-		return
-	}
-	log.WithField(configs.Component, "Logger Init").Infof("Logging configuration: %+v", config)
-
-	level, err := log.ParseLevel(strings.ToLower(config.Level))
+	level, err := log.ParseLevel(strings.ToLower(logLevel))
 	if err != nil {
-		log.WithField(configs.Component, "Logger Init").Error(err)
```
Comment (AntiD2ta) [inline cli/root.go:134]: Why did this log got removed?

File: cli/root.go [hunk 13]
```diff
@@ -114,24 +70,14 @@ returns :-
...
-		FieldsOrder:     []string{configs.Component},
 		TimestampFormat: "2006-01-02 15:04:05 --",
 	})
 
-	err := viper.UnmarshalKey("logs", &config)
-	if err != nil {
-		log.WithField(configs.Component, "Logger Init").Errorf("Unable to decode into struct, %v", err)
-		return
-	}
-	log.WithField(configs.Component, "Logger Init").Infof("Logging configuration: %+v", config)
```
Comment (AntiD2ta) [inline cli/root.go:130]: Why did this log got removed?

File: cli/root.go [hunk 14]
```diff
@@ -48,50 +44,10 @@ func RootCmd() *cobra.Command {
 	// Disable completion default cmd
 	cmd.CompletionOptions.DisableDefaultCmd = true
 	// Persistent flags
-	cmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.sedge.yaml)")
+	cmd.PersistentFlags().StringVar(&logLevel, "logLevel", "info", "Set Log Level")
```
Comment (AntiD2ta) [inline cli/root.go:47]: The help message should be:
```
Set Log Level, e.g. info, debug, error, ...
```

Put all the possible values there ^^^

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
Very helpful PR, minor changes requested including:

- [x] Update `SupportedClients` description
- [ ] Update `Clients` description
- [ ] Remove prysm.tmpl from Gnosis envs templates

File: docs/docs/commands/clients.mdx (hunk 8 above)
Comment (stdevMac) [reply docs/docs/commands/clients.mdx:48]: for the consensus is available

File: docs/docs/commands/clients.mdx (hunk 9 above)
Comment (stdevMac) [reply docs/docs/commands/clients.mdx:35]: solved at 6e38108

File: docs/docs/commands/cli.mdx (hunk 10 above)
Comment (stdevMac) [reply docs/docs/commands/cli.mdx:71]: solved at 6e38108

File: cli/root.go (hunk 11 above)
Comment (stdevMac) [reply cli/root.go:121]: solved at 6e38108

File: cli/root.go (hunk 12 above)
Comment (stdevMac) [reply cli/root.go:134]: solved 6e38108

File: cli/root.go (hunk 13 above)
Comment (stdevMac) [reply cli/root.go:130]: solved at 6e38108

File: cli/root.go (hunk 14 above)
Comment (stdevMac) [reply cli/root.go]: solved at 6e38108

File: docs/docs/commands/down.mdx (hunk 7 above)
Comment (stdevMac) [reply docs/docs/commands/down.mdx:34]: solved at 6e38108

File: docs/docs/commands/keys.mdx (hunk 6 above)
Comment (stdevMac) [reply docs/docs/commands/keys.mdx:66]: solved at 6e38108

File: docs/docs/commands/logs.mdx (hunk 5 above)
Comment (stdevMac) [reply docs/docs/commands/logs.mdx:38]: solved at 6e38108

File: docs/docs/commands/networks.mdx (hunk 4 above)
Comment (stdevMac) [reply docs/docs/commands/networks.mdx:34]: solved at 6e38108

File: docs/docs/commands/version.mdx (hunk 3 above)
Comment (stdevMac) [reply docs/docs/commands/version.mdx:34]: solved at 6e38108

File: internal/pkg/clients/clients_test.go (hunk 2 above)
Comment (stdevMac) [reply internal/pkg/clients/clients_test.go]: solved at 6e38108

File: internal/pkg/clients/clients.go (hunk 1 above)
Comment (stdevMac) [reply internal/pkg/clients/clients.go]: solved at 6e38108

File: CHANGELOG.md [hunk 15]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Removed
```
[the remark this answers is not in the document]
Comment (stdevMac) [reply CHANGELOG.md]: solved at 6e38108

File: internal/pkg/clients/clients_test.go (hunk 2 above)
Comment (AntiD2ta) [reply internal/pkg/clients/clients_test.go]: I don't know how is that test not failing, gnosis doesn't support `prysm`. Check that again

File: cli/root.go [hunk 16]
```diff
@@ -48,50 +44,10 @@ func RootCmd() *cobra.Command {
 	// Disable completion default cmd
 	cmd.CompletionOptions.DisableDefaultCmd = true
 	// Persistent flags
-	cmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.sedge.yaml)")
+	cmd.PersistentFlags().StringVar(&logLevel, "logLevel", "info", "Set Log Level, e.gg panic, fatal, error, warn, warning, info, debug. trace")
```
Comment (AntiD2ta) [inline cli/root.go:47]:
```suggestion
	cmd.PersistentFlags().StringVar(&logLevel, "logLevel", "info", "Set Log Level, e.g panic, fatal, error, warning, info, debug, trace")
```

File: docs/docs/commands/clients.mdx (hunk 8 above)
Comment (AntiD2ta) [reply docs/docs/commands/clients.mdx]: One of the requested changes was: Remove prysm.tmpl from Gnosis envs templates

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
3 more comments to go

File: internal/pkg/clients/clients_test.go (hunk 2 above)
Comment (stdevMac) [reply internal/pkg/clients/clients_test.go]: Check here please 6d2386a37497b6c648acfc3ec9e6132af8b67748

File: cli/root.go (hunk 16 above)
Comment (stdevMac) [reply cli/root.go]: Solved here f91852e

**Comment by stdevMac** [comment]
@AntiD2ta please review again, all the issues you found, should be solved now

**Review by AntiD2ta — APPROVED** [review]

**Pull request merged**